src/server/game/Server/WorldSocket.cpp

```diff
@@ -358,6 +358,8 @@ WorldSocket::ReadDataHandlerResult WorldSocket::ReadDataHandler()
         }
         case CMSG_KEEP_ALIVE:
             LogOpcodeText(opcode, sessionGuard);
+            if (_worldSession)
```

Comment:
@jackpoz the type of sessionGuard is unique_lock (constructed with deferred lock flag), there is no need to unlock it